examples/rbd/storageclass.yaml

```diff
@@ -36,6 +36,11 @@ parameters:
    csi.storage.k8s.io/fstype: ext4
    # uncomment the following to use rbd-nbd as mounter on supported nodes
    # mounter: rbd-nbd
+
+   # Instruct the plugin it has to encrypt the volume
+   # By default it is disabled. Valid values are “true” or “false”.
+   # A string is expected here, i.e. “true”, not true.
+   # encrypted: "true"
```

Comment:
if you check existing StorageClass for [AWS EBS](https://kubernetes.io/docs/concepts/storage/storage-classes/#aws-ebs) you can see that it is using also `encrypted` field rather than image features, so I think it's more natural to expose it in a similar way.

Reply:
Yes, I agree and I have seen that. They also lack (or do not need) an `imageFeatures` parameter, which we could leverage.

This is just a thought, as I said, I am more for reducing the StorageClass parameters, but let's hear from others and see if this makes sense in this particular case.

Reply:
I'd prefer to avoid overloading image features. In a perfect world, honestly, the CSI should just manage them and a user should never touch them. We want to get to the point where users of the CSI really don't need to understand how RBD/CephFS work internally.

Reply:
+1 @dillaman The k8s user does not need to know about the Ceph. But I agree too @ShyamsundarR , we could reduce the StorageClass parameters as much possible. I think the CSI interface has to accept more arguments like 'encryption'. (and kubernetes itself maybe?)

But for now, we still need a new variable in the StorageClass, and i'd prefer the variable name is not 'imageFeatures'. The word is using in the Ceph, and it could be confused. (a user doesn't want to know about the Ceph word...)

Reply:
Ack! thanks for the feedback. We will proceed without overloading the Ceph `imageFeatures` term, and use separate parameters for now (till other systemic improvements provide a way to optimize parameters in the StorageClass).